modules/ossm-config-otel.adoc

```diff
@@ -70,7 +70,7 @@ spec:
     meshConfig:
       enableTracing: true
       extensionProviders:
-      - name: otel-tracing
+      - name: otel
```

Comment:
A small observation:

In the prerequisites section (Line 11), there is no mention about the following. 

```
OpenShift Service Mesh (OSSM) Operator is installed. See: [xxxxxxx]
```

If you think its already captured in the parent links which redirected the user here (or its optional because the doc is about OSSM), then please ignore my comment.

Reply:
In this particular context, I think we're ok not calling out the Operator. I think it's addressed by the parent topics.